### PR TITLE
E2E test menu

### PR DESCRIPTION
**Pull request by guyyoo**

### CODE REVIEW

File: kitchenSink.json [hunk 1]
```diff
@@ -8815,9 +8815,9 @@
                     {
                         "language_code": "en",
                         "label": "Menu.menu",
-                        "menuTextLeft": "fdgdfg",
-                        "menuTextCenter": "fdgdfg",
-                        "menuTextRight": "fdgdfg"
+                        "menuTextLeft": "Text1",
+                        "menuTextCenter": "Text2",
+                        "menuTextRight": "Text3"
```
Comment (nh758) [inline kitchenSink.json:8820]: I like this change!

File: test_cases/Menu.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
+export default (folderName, Common) => {
+    describe("Menu", () => {
+      it("Exists", () => {
+        cy.get(
+          '[data-cy="tab-Menu-773dca98-87e0-4dc2-b528-89ec7c98c448-b52e6e96-5033-4c7f-a104-29bd5ddcac4a"]'
+        )
+          .click();
```
Comment (nh758) [inline test_cases/Menu.js:7]: You do this in each test. Please move into a BeforeEach hook

File: test_cases/Menu.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+        cy.get(
+          '[data-cy="menu-item Chart 41d39ac7-e7d9-405d-a570-b79686ccdd5a eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+        )
+          .click();
+
+        cy.get('div[view_id^="ABViewChartPie_"]')
+          .should("include.text", "7")
```
Comment (nh758) [inline test_cases/Menu.js:65]: I think you can simplify this to `.should('exist')`. We don't want to test the Chart Widget here.

File: test_cases/Menu.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+          .should("include.text", "Home")
+        
+        cy.get('[data-cy="menu-item 2 398a96a6-21dc-4ab3-8017-b2754fa979d4 5a50210f-767b-461e-9762-66fbc263ec7f"]')
+          .should("be.visible")
+          .should("include.text", "2")
+
+
+        cy.get(
+          '[data-cy="menu-item 2 398a96a6-21dc-4ab3-8017-b2754fa979d4 5a50210f-767b-461e-9762-66fbc263ec7f"]'
+        )
```
Comment (nh758) [inline test_cases/Menu.js:94]: I don't think you need to get the element again. Can move the `click()` to line 90

File: test_cases/Menu.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+          .click();
+
+        cy.get(
+          '[data-cy="menu-item 1 93f39714-7d1b-4375-b5b2-89c94a3e96bd eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+        )
+          .click();
+
+        cy.get('[data-cy="menu-item KitchenHome cb77ced0-a803-46b7-8a79-f9084d75d51c 5a50210f-767b-461e-9762-66fbc263ec7f"]')
+          .should("be.visible")
+          .should("include.text", "Home")
```
Comment (nh758) [inline test_cases/Menu.js:85]: missing `;` and some other places. Maybe run eslint/prettier?

**Review by nh758 — CHANGES_REQUESTED** [review]
Good start. A few things to fix.

**Review by nh758 — APPROVED** [review]
Looks good.

**Comment by cmboyman** [comment]
pretty good

**Review by cmboyman — APPROVED** [review]
very good.

**Pull request merged**